doc/learning-to-rank.md

```diff
@@ -153,6 +153,140 @@ work with this Concourse and SageMaker configuration.
    repository is given in the output of deploying the `app-search`
    terraform.
 
+### Training and deploying a second model for A/B testing
+
```

Comment:
It would great to add a description for this section below the heading, describing what the result of this process would be and why we'd want to generate extra models for AB testing.

I reckon more general docs about our AB testing approach for search would be useful actually, but it's out of scope here.